devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy

```diff
@@ -150,6 +151,15 @@ def parse(String description) {
 				state.saturationValue = Math.round(zigbee.convertHexToInt(zigbeeMap.value) / 0xfe * 100)
 				runIn(5, updateColor, [overwrite: true])
 			}
+			else if (zigbeeMap.attrInt == ATTRIBUTE_COLOR_MODE){ //Color Mode Attribute
```

Comment:
Since this didn't work I think it's best to remove it

Reply:
Sure, done!